docker/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+FROM mcr.microsoft.com/dotnet/sdk:3.1-alpine
+WORKDIR /opt/amt
+COPY . .
```

Comment:
There is no need for a `Dockerfile` in this story. We don't need to build any new containers. We just need to run PostgreSQL and SQL Server.